phabfive/maniphest.py

```diff
@@ -124,9 +289,15 @@ def create_from_config(self, config_file, dry_run=False):
             transactions = [
                 {"type": "title", "value": ticket_data["title"]},
                 {"type": "description", "value": ticket_data["description"]},
-                {"type": "priority", "value": ticket_data.get("priority", TICKET_PRIORITY_NORMAL)},
             ]
 
+            if ticket_data.get("priority"):
+                transactions.append({
+                    "type": "priority", "value": ticket_data["priority"]},)
```

Comment:
```suggestion
                transactions.append({"type": "priority", "value": ticket_data["priority"]},)
```

Line wrapping here is not needed, we abide by ~160 character width in general and unless you have much nested data, it can be on a single line